### PR TITLE
Raise when table name not camel-cased in gen tasks

### PR DESCRIPTION
**Pull request by gjaldon**
Closes https://github.com/phoenixframework/phoenix/issues/1191

### CODE REVIEW

File: lib/mix/tasks/phoenix.gen.html.ex [hunk 1]
```diff
@@ -66,10 +66,13 @@ defmodule Mix.Tasks.Phoenix.Gen.Html do
...
   defp validate_args!([_, plural | _] = args) do
-    if String.contains?(plural, ":") do
-      raise_with_help
-    else
-      args
+    cond do
+      String.contains?(plural, ":") ->
+        raise_with_help
+      plural != Phoenix.Naming.underscore(plural) ->
+        Mix.raise "expected the second argument, `#{plural}`, to be all lowercase using snake_case convention"
```
Comment (josevalim) [inline lib/mix/tasks/phoenix.gen.html.ex]: I don't think we use backticks in error messages in phoenix. Can you please use `inspect plural` instead?

**Comment by josevalim** [comment]
One tiny change left and we are good to go! :D

**Comment by gjaldon** [comment]
Updated the PR! My pleasure :)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**